### DISCUSSION TITLE
Remover Gerenciador de Colunas de um Page Dynamic Table

### DISCUSSION THREAD

**Discussion by gtrabach** [po-ui/po-angular#1781] [Tire suas dúvidas] [answered]
Bom dia!

Existe alguma forma de remover o botão de gerenciador de colunas de um Page Dynamic Table? Objetivo: não exibir o botão, assim, não permitindo o usuário alterar a visibilidade e ordem dos campos.

![gerenciador_colunas_poui](https://github.com/po-ui/po-angular/assets/46979335/743fe729-7028-4f4c-a47c-ee120dae3712)

**Comment by jhosefmarks** [comment]
Use a propriedade `p-hide-columns-manager`.

Conforme a doc: https://po-ui.io/documentation/po-table

> Permite que o gerenciador de colunas, responsável pela definição de quais colunas serão exibidas, seja escondido.

**Reply by gtrabach** [reply]
Boa tarde, Jhosef. 

Como posso utilizar essa propriedade na Dynamic Table? Quando adiciono ela é gerado o seguinte erro no meu projeto: `Can't bind to 'p-hide-columns-manager' since it isn't a known property of 'po-page-dynamic-table'.`

Quando adiciono sem colchetes, não é efetuada nenhuma alteração na tabela.

Ao adicionar em uma Table comum ele funciona perfeitamente.

**Reply by jhosefmarks** [reply, marked as answer]
Talvez precise fazer o repasse internamente no componente. Ai a sugestão é vc abrir uma issue para o time de desenvolvimento.

**Comment by jkozima** [comment]
Bom dia!
A correção foi aplicada na versão 16.3.0 do PO UI.
Obrigada.
